docs/geedocs/5.2.2/index.html

```diff
@@ -40,7 +40,7 @@ <h4><a href="answer/2987876.html">Legal notices</a></h4>
   </div>
```

Comment:
Should 'Release notes' points to a new .html file (may be 716002.html) to be created for hosting 5.2.2 release notes ?

Reply:
I could do that here, but normally release notes is done as a separate task. Do you mean just to create a "skeleton" or blank page to be filled out later?

Reply:
yeah, a skeleton would be fine..